install/kubernetes/helm/istio/charts/kiali/templates/serviceaccount.yaml

```diff
@@ -8,7 +8,7 @@ imagePullSecrets:
 {{- end }}
 {{- end }}
 metadata:
-  name: kiali-service-account
+  name: kiali
```

Comment:
This should not be changed.

Reply:
@gbaufake this is the change to "kiali" that needs to be reverted